lib/google_scraping/account/guardian.ex

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleScraping.Account.Guardian do
+  use Guardian, otp_app: :google_scraping
+
+  alias GoogleScraping.Accounts
+
+  def subject_for_token(user, _claims), do: {:ok, to_string(user.id)}
+
+  def resource_from_claims(%{"sub" => id}) do
```

Comment:
> Accounts.get_user!

Shall we use `!`(`resource_from_claims!`) as this function can raise exception

Reply:
It was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/54/commits/ea73addd17c55213e3ce510e0820b7d8c675d328